R/rename.R

```diff
@@ -1,8 +1,8 @@
 polish_emissions_profile_upstream_company <- prepare_ictr_company
-polish_emissions_profile_company <- prepare_pctr_company
 polish_emissions_profile_upstream_product <- prepare_ictr_product
+polish_sector_profile_upstream_company <- prepare_istr_company
+polish_sector_profile_upstream_product <- prepare_istr_product
+polish_emissions_profile_company <- prepare_pctr_company
 polish_emissions_profile_product <- prepare_pctr_product
-polish_sector_profile_product <- prepare_pstr_product
 polish_sector_profile_company <- prepare_pstr_company
-polish_sector_profile_upstream_product <- prepare_istr_product
-polish_sector_profile_upstream_company <- prepare_istr_company
+polish_sector_profile_product <- prepare_pstr_product
```

Comment:
This is just refactoring to sort the functions alphabetically. Not strictly in scope but I guess it's fine.